container_test.go

```diff
@@ -1091,10 +1093,7 @@ func TestAttachToContainerRawTerminalFalse(t *testing.T) {
 		Stream:       true,
 		RawTerminal:  false,
 	}
-	err := client.AttachToContainer(opts)
-	if err != nil {
-		t.Fatal(err)
-	}
+	client.AttachToContainer(opts)
```

Comment:
I noticed while editing it, just fixed the test.

Also changed the code to use Docker's StdCopy (it's now exported there).